lib/pbench/cli/agent/commands/results/push.py

```diff
@@ -24,10 +25,26 @@ def execute(self) -> int:
             self.config,
             self.logger,
         )
-        crt.copy_result_tb(
+        res = crt.copy_result_tb(
             self.context.token, self.context.access, self.context.metadata
         )
-        return 0
+
+        try:
+            msg = res.json()["message"]
+        except requests.exceptions.JSONDecodeError:
+            msg = "FOO"
+
+        if res.ok:
+            click.echo(
+                f"Status: {res.status_code}, reason: {res.reason}, message: {msg}"
+            )
+            return 0
+        else:
+            click.echo(
+                f"Error status: {res.status_code}, reason: {res.reason}, message: {msg}",
+                err=True,
+            )
```

Comment:
I don't mind writing the HTTP `status_code`, although it's not entirely user friendly; but we should probably clarify that it's the HTTP status rather than some obscure internal error number. And while it's "cute" to say that 400 is "Bad request" I'm not sure it's really that useful to have both `status_code` and `reason`. If we get a `message` from the server JSON that's much more specific and useful, and the simplistic HTTP `reason` is redundant: if we don't get a message, then `reason` (to me) is more of a last-ditch message fallback.

Reply:
OK